README.md

```diff
@@ -79,8 +79,7 @@ require('yargs') // eslint-disable-line
     serve(argv.port)
   })
   .option('verbose', {
-    alias: 'v',
-    default: false
+    alias: 'v'
```

Comment:
perhaps add a `describe: 'run with verbose logging'`, I'm comfortable with dropping the `false` 👍

Reply:
If we want this example to show a standard way of defining a verbose option, I suggest also adding a `type: 'count'`, as it is the way a verbose option often works (v = 0 when the option is absent, 1 when it is present, up to n when it is repeated, 4 for example with `-vvvv`).

Reply:
I was thinking of suggesting this too, but thought it might be a bit too advanced of a feature to show off near the top of the README. I'm comfortable with whichever approach @sliftist takes 👍 but think it would be good to add a `description` either way.

Reply:
Cool, I didn't know `type: 'count'` worked like that. Both changes added to the pull request.